src/makeInfinityAdder.js

```diff
@@ -6,7 +6,23 @@
  */
 
 function makeInfinityAdder() {
-  // write code here
+  let sum = 0;
+
+  const adder = (...value) => {
+    if (value.length) {
+      sum += value[0];
+
+      return adder;
+    }
```

Comment:
Is it necessary to rest params?
Your function should take 1 argument.

```suggestion
  const adder = (number) => {
    if (some cond) {
      sum += number;

      return adder;
    }
```